tests/cases/test_titan.rs

```diff
@@ -148,6 +149,21 @@ fn test_titandb() {
         }
     }
 
+    let mut readopts = ReadOptions::new();
+    readopts.set_titan_key_only(true);
+    iter = db.iter_opt(readopts);
+    iter.seek(SeekKey::Start);
+    for i in 0..n {
+        for j in 0..n {
+            let k = (i * n + j) as u8;
+            let v = vec![k; (j + 1) as usize];
+            assert_eq!(db.get(&[k]).unwrap().unwrap(), &v);
+            assert!(iter.valid());
+            assert_eq!(iter.key(), &[k]);
```

Comment:
check if value is empty?

Reply:
We can't, cause `value()` will lead to `assert(options_.key_only)` to abort. There is a way of catching panic in rust —`catch_unwind()`, but I can't make it catch the abort from C++, do you have any idea?

Reply:
ok then.